Userland/Libraries/LibWeb/Page/Frame.cpp

```diff
@@ -305,4 +305,18 @@ void Frame::unregister_viewport_client(ViewportClient& client)
     VERIFY(was_removed);
 }
 
+void Frame::register_frame_nesting(URL url)
+{
+    if (!m_frame_nesting_levels.contains(url))
+        m_frame_nesting_levels.set(url, 0);
+    (m_frame_nesting_levels.find(url)->value)++;
+}
+
+bool Frame::is_frame_nesting_allowed(URL url)
+{
+    if (m_frame_nesting_levels.contains(url) && m_frame_nesting_levels.find(url)->value >= max_recursion_depth)
+        return false;
+    return true;
```

Comment:
We can use `HashMap::get()` here to simplify this function:
```cpp
return m_frame_nesting_levels.get(url).value_or(0) >= max_recursion_depth;
```

Reply:
This PR turns into a lesson of HashMap methods :) Thanks for pointing them out!